client/app/components/packages/scope-of-work-draft/edit.hbs

```diff
@@ -0,0 +1,97 @@
+<SaveableForm 
```

Comment:
why is `draft` at the end of the folder title here?
Why not follow the previous pattern of `draft-scope-of-work/edit.hbs`?
Like `draft-eas/edit.hbs`?

Reply:
@godfreyyeung , this is suggested by @allthesignals